libmscore/system.cpp

```diff
@@ -39,6 +39,8 @@
 #include "chordrest.h"
 #include "iname.h"
 #include "spanner.h"
+#include "sym.h"
+#include "symbol.h"
```

Comment:
`#include "symbol.h"` is probably not needed since it's already included in system.h ?